src/datamodel_code_generator/_types/generate_config_dict.py

```
@@ -5,7 +5,7 @@
 
 from typing import TYPE_CHECKING, Any, TypedDict
 
-from typing_extensions import NotRequired
+from typing_extensions import NotRequired, TypedDict
```

Ruff (0.14.10)

Redefinition of unused `TypedDict` from line 6

(F811)